src/EditorFeatures/Core/Shared/Extensions/TaggedTextExtensions.cs

```diff
@@ -71,12 +71,17 @@ public static TextBlock ToTextBlock(this TaggedText part, ClassificationTypeMap
         public static TextBlock ToTextBlock(
             this IEnumerable<Inline> inlines,
             ClassificationTypeMap typeMap,
-            string classificationFormatMap = null)
+            string classificationFormatMap = null,
+            bool wrap = true)
         {
             classificationFormatMap = classificationFormatMap ?? "tooltip";
             var formatMap = typeMap.ClassificationFormatMapService.GetClassificationFormatMap(classificationFormatMap);
 
-            var textBlock = new TextBlock { TextWrapping = TextWrapping.Wrap };
+            var textBlock = new TextBlock
+            {
+                TextWrapping = wrap ? TextWrapping.Wrap : TextWrapping.NoWrap,
+                TextTrimming = wrap ? TextTrimming.None : TextTrimming.CharacterEllipsis
+            };
```

Comment:
This doesn't warrant it to me as this is just as a normal expression statement with an assignment in it.  If the statement ended with ```}``` i'd feel differently.